src/hotspot/os/windows/os_windows.cpp

```diff
@@ -4819,6 +4819,8 @@ int os::stat(const char *path, struct stat *sbuf) {
     DWORD errcode = ::GetLastError();
     if (errcode == ERROR_FILE_NOT_FOUND || errcode == ERROR_PATH_NOT_FOUND) {
       errno = ENOENT;
+    } else {
+      _set_errno(0);
```

Comment:
I don't object to setting it to zero explicitly but we do not need to use `_set_errno` to do it!
```suggestion
      errno = 0;
```

Reply:
Addressed.